docs/dataset/advanced-usage/upload-embeddings.md

```diff
@@ -0,0 +1,65 @@
+---
+icon: kolena/classification-16
+---
+
+# :kolena-classification-16: Uploading Custom Embeddings
+
+This guide explains how to upload your own embeddings to Kolena using the Kolena SDK.
+Please ensure you have the SDK installed.
+[Instructions for installing the SDK are available here.](https://docs.kolena.com/installing-kolena/)
+
+## Step 1: Import the Embedding Upload Function
+
+To upload embeddings, use the `upload_dataset_embeddings` function from Kolena. You can import
+it with the following code:
+```
```

Comment:
let's do ``` python to the syntax highlighting can show:
<img width="688" alt="image" src="https://github.com/user-attachments/assets/25db3b9b-d0ba-4b6f-87bb-0452a8211198">

Reply:
Didn't know this was a thing. Updated the docs accordinly